app/controllers/middleware/errorMiddleware.js

```diff
@@ -0,0 +1,4 @@
+exports.errorMiddleware = (error, req, res, next) => {
+  console.log('Error middleware', error);
+  res.send('Error');
```

Comment:
I am not familiarised with the integration with slack but I am wondering if errors status code should be `500` instead